indy_node/server/domain_req_handler.py

```diff
@@ -214,6 +200,40 @@ def _validateAttrib(self, req: Request):
                 "Only identity owner/guardian can add attribute "
                 "for that identity")
 
+    def _validate_schema(self, req: Request):
+        # we can not add a Schema with already existent NAME and VERSION
+        # sine a Schema needs to be identified by seqNo
+        identifier = req.identifier
+        operation = req.operation
+        schema_name = operation[DATA][NAME]
+        schema_version = operation[DATA][VERSION]
+        schema, _, _, _ = self.getSchema(
+            author=identifier,
+            schemaName=schema_name,
+            schemaVersion=schema_version
+        )
+        if schema:
+            raise InvalidClientRequest(identifier, req.reqId,
+                                       '{} can have one and only one SCHEMA with '
+                                       'name {} and version {}'
+                                       .format(identifier, schema_name, schema_version))
+
+    def _validate_claim_def(self, req: Request):
+        # we can not add a Claim Def with existent ISSUER_DID
+        # sine a Claim Def needs to be identified by seqNo
+        identifier = req.identifier
+        operation = req.operation
+        schema_ref = operation[REF]
+        claim_def, _, _, _ = self.getClaimDef(
+            author=identifier,
+            schemaSeqNo=schema_ref
+        )
+        if claim_def:
+            raise InvalidClientRequest(identifier, req.reqId,
+                                       '{} can have one and only one CLAIM_DEF with '
+                                       'issuer DID (identifier) {} and schema ref {}'
```

Comment:
This should read 'signature scheme {} and schema ref {}'

Reply:
Done